third_party/move/move-model/src/demove_helper.rs

```diff
@@ -0,0 +1,133 @@
+use std::{collections::BTreeMap, rc::Rc, borrow::Borrow};
```

Comment:
This doesn't belong in the general move-model code but should go into the decompiler tree, or as I suggested, go into an existing module. You can't have `myfeature_helpers.rs` in the model of generic framework code.